src/main/java/uk/gov/beis/els/categories/spaceheaters/model/BoilerPackagesCalculatorForm.java

```diff
@@ -15,19 +15,19 @@
 
   @FieldPrompt("Does this package include a supplementary heat pump?")
   @NotNull(message = "Specify if the package includes a supplementary heat pump")
-  private Boolean hasSupplementaryHeatPump;
+  private Boolean supplementaryHeatPump;
```

Comment:
All the other boolean fields are just `foo` rather than `hasFoo` so I've made them consistent. Arguably `hasFoo` is a bit nicer but it would require a lot more refactoring than doing it this way round and I think as long as its consistent either is fine.